tecken/tests/test_upload.py

```diff
@@ -933,6 +933,28 @@ def test_clearuploads_records_dry_run(db, fakeuser):
 
 
 class Test_remove_orphaned_files:
+    def test_no_watchdir(self, db, settings, caplog):
+        caplog.set_level(logging.INFO)
+
+        tempdir = "/does/not/exist"
+        settings.UPLOAD_TEMPDIR = tempdir
+        settings.UPLOAD_TEMPDIR_ORPHANS_CUTOFF = 5
+
+        call_command("remove_orphaned_files", verbose=True)
+
+        # Make sure we've got the right expires value
+        assert "expires: 5 (minutes)" in caplog.text
```

Comment:
Huh, I'm surprised that the tests in this file are making assertions based on what's logged rather than what functions are/are not called or what the function under test returns. Why is it set up this way?

It seems like the log line/branching logic could stay the same, but some substantive part of the function body could change (say a function is no longer called that previously was), and these tests would still pass. 🤔